src/node/internal/zlib.d.ts

```diff
@@ -1,14 +1,10 @@
 import { owner_symbol, type Zlib } from 'node-internal:internal_zlib_base';
 
-export function crc32(data: ArrayBufferView, value: number): number;
-
-export type CompressCallback = (
-  err: Error | null,
-  buffer?: ArrayBuffer
-) => void;
-export type InternalCompressCallback = (res: Error | ArrayBuffer) => void;
+type InternalCompressCallback = (res: Error | ArrayBuffer) => void;
 
+export function crc32(data: ArrayBufferView, value: number): number;
```

Comment:
The next line cover this already, why did we need this line?

Reply:
Must've been a copy paste accident; nuked